getting_started/srun.md

```diff
@@ -0,0 +1,52 @@
+## `srun`: Submitting Jobs to the Slurm Workload Manager
+
+Slurm is a job scheduling system commly used in High-Performance Computing (HPC) environments to efficiently allocate and manage computing resources among users.
+`srun` is a command-line tool used for submitting jobs to the Slurm workload manager, offering a wide range of options for customizing job submission.
+With `srun`, users can easily launch jobs that require multiple CPU cores, large amounts of memory, and long runtimes, while taking advantage of the scalability and reliability of the Slurm workload manager.
+
+### `srun` examples
+
+The basic syntax for submitting a job with `srun` is as follows:
+
+```
+srun [options] command
+```
+
+where `command` is the command or script to be executed. 
+
+To request one node, with 2GB of memory, for 30 minutes, on the `cpu` partition, type:
+
+```
+srun --partition=cpu --nodes=1 --mem=2G --time=30 --pty bash
+```
+
+Note:
+The option `--pty` here is essential.
+You cannot enter the srun environment without `--pty`.
+
+You can also get notification by email when the 'srun' environment is ready if you want.
```

Comment:
I'm not sure if "`srun` environment" is the right choice of words. I would change it to "interactive job".